evap/results/templates/results_evaluation_detail.html

```diff
@@ -26,43 +26,55 @@
-                    {% trans 'Full' %}
-                    </a>
-                    {% if not evaluation.can_publish_rating_results %}
-                        <button type="button" disabled class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="bottom"
-                            title="{% blocktrans %}The results of this evaluation have not been published because it didn't get enough votes.{% endblocktrans %}">
-                            {% if evaluation.course.is_private %}
-                                {% trans 'Participant' %}
-                            {% else %}
-                                {% trans 'Public' %}
-                            {% endif %}
-                        </button>
-                    {% else %}
-                        <a href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=public" role="button" class="btn btn-sm btn-light{% if view == 'public' %} active{% endif %}" data-bs-toggle="tooltip" data-bs-placement="bottom"
-                           title="
+                        {% if not evaluation.can_publish_rating_results %}
+                            <button type="button" disabled class="btn btn-sm btn-light" data-bs-toggle="tooltip"
+                                    data-bs-placement="bottom"
+                                    title="{% blocktrans %}The results of this evaluation have not been published because it didn't get enough votes.{% endblocktrans %}">
+                                {% if evaluation.course.is_private %}
+                                    {% trans 'Participant' %}
+                                {% else %}
+                                    {% trans 'Public' %}
+                                {% endif %}
+                            </button>
+                        {% else %}
+                            <a href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=public"
+                               role="button" class="btn btn-sm btn-light{% if view == 'public' %} active{% endif %}"
+                               data-bs-toggle="tooltip" data-bs-placement="bottom"
+                               title="
                         {% if evaluation.course.is_private %}
                             {% trans 'Shows results available for the participants.' %}">
-                                {% trans 'Participant' %}
-                        {% else %}
-                            {% trans 'Shows results available for everyone logged in.' %}">
-                                {% trans 'Public' %}
+                                   {% trans 'Participant' %}
+                               {% else %}
+                                   {% trans 'Shows results available for everyone logged in.' %}">
+                                   {% trans 'Public' %}
+                               {% endif %}
+                            </a>
```

Comment:
I think the indentation of the django tags should be "correct" following their django semantics, i.e. the `if` here needs to be indented more and the `else` and `endif` should be on the same indentation level as the `if`. The current indentation confused me a lot, I initially thought the `</a>` was moved inside the `if` accidentally.

I do see that the code here is confusing in any formatting because we have control logic while opening the anchor tag. The intention here was to not duplicate all the other attributes. This might be cleaner if expressed using [`blocktranslate`](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#blocktranslate-template-tag) with the `asvar` attribute to create two new "local variables" in the template that can then be used as title and content of the anchor. If you want, you can change that.